utils/api/base.js

```diff
@@ -12,7 +12,9 @@ export const fetcher = (...args) => {
 
 export const posting = async (url, data) => {
   try {
-    await api.post(url, data)
+    const response = await api.post(url, data)
+    // statusText: "Created"
+    console.log({ response })
```

Comment:
keeping this console.log here for now since work will be continuing on this file in order to add a redirect